doc/connection-backoff.md

```diff
@@ -32,11 +32,14 @@ ConnectWithBackoff()
 
 With specific parameters of
 MIN_CONNECT_TIMEOUT = 20 seconds
-INITIAL_BACKOFF = 1 second
-MULTIPLIER = 1.6
+INITIAL_BACKOFF = 1.170820393 seconds
+MULTIPLIER = 1.618033989
 MAX_BACKOFF = 120 seconds
 JITTER = 0.2
 
+These parameters make each successive delay equal to the sum of the two previous
+delays (ignoring jitter and max backoff).
+
```

Comment:
Don't want to beat a dead horse, but not sure why the above paragraph is a good idea. If it is a good idea, though, the implementation should use integer addition rather than float addition/multiplication since it could be faster.